docs/030_user-guide/030_actions/010_mutate.md

```diff
@@ -78,6 +78,34 @@ When(a.ConfigMap)
   });
 ```
 
+## Mutate Alias
+
+You can also use the Alias function to include a user-defined alias for a Mutate action in the logs for easier debugging. This is especially useful when you have multiple Mutate actions in a single module.
+
+For example, to add an alias to a Mutate action:
+
+```typescript
+When(a.Pod)
+  .IsCreatedOrUpdated()
+  .InNamespace("pepr-demo")
+  .WithLabel("pepr-test-pod")
+  .Alias("reject:pods:runAsRoot:privileged:runAsGroup:allowPrivilegeEscalation")
+  .Mutate(async (po, logger) => {
+    logger.info(`Pod ${po.Raw.metadata?.name} is being mutated.`);
+  });
```

Comment:
Add Watch Example too since it will not be `po.Raw` in this case, will be good to see.